REQUIRE

```diff
@@ -11,3 +11,4 @@ StatsPlots
 Flux#master
 StatsFuns
 GLM
+NNlib
```

Comment:
Could we just replace the REQUIRE file with a Project.toml file? REQUIRE files were replaced with Project.toml files quite a while ago.

Reply:
Done!